l5kit/l5kit/tests/rasterization/box_rasterizer_test.py

```diff
@@ -78,5 +78,5 @@ def test_out_shape(hist_data: tuple, dmg: LocalDataManager, cfg: dict) -> None:
 
     rasterizer = build_rasterizer(cfg, dmg)
 
-    out = rasterizer.rasterize(hist_data[0][: hist_length + 1], hist_data[1][: hist_length + 1])
+    out = rasterizer.rasterize(hist_data[0][: hist_length + 1], hist_data[1][: hist_length + 1], [])  # TODO TR_FACES
```

Comment:
Let's remove these TODO. Let's add the test (ideally), or create an issue.

Reply:
we cannot add it ass our artefacts don't have TF yet, but we can remove it from here as the BoxRasterizer is not affected by it anyway